document/js-api/index.bs

```diff
@@ -0,0 +1,840 @@
+    1. Set |memory|.\[[BufferObject]] to |buffer|.
+    1. Return |ret|.
+</div>
+
+<h3 id="tables">Tables</h3>
+
+<pre class="idl">
+enum TableKind {
+  "anyfunc",
+};
+
+dictionary TableDescriptor {
+  required TableKind element;
+  required [EnforceRange] unsigned long initial;
+  [EnforceRange] unsigned long maximum;
+};
+
+[LegacyNamespace=WebAssembly, Constructor(TableDescriptor descriptor), Exposed=(Window,Worker,Worklet)]
+interface Table {
+  void grow([EnforceRange] unsigned long delta);
+  Function? get([EnforceRange] unsigned long delta);
+  void set([EnforceRange] unsigned long delta, Function? value);
+  readonly attribute unsigned long length;
+};
+</pre>
+
+<div>
+A {{Table}} object represents a single [=table instance=]
+which can be simultaneously referenced by multiple {{Instance}} objects. Each
+{{Table}} object has one internal slots:
+
+    * \[[Table]] : a [=table address=]
+    * \[[Values]] : a List whose elements are either null or [=Exported Function=]s.
+</div>
+
+<div algorithm>
+    To <dfn>create a table object</dfn> from a [=table address=] |tableaddr|, perform the following steps:
+    1. Let |values| be a list whose length is [=size_table=](|store|, |tableaddr|) where each element is null.
+    1. Return a new {{Table}} instance with \[[Table]] set to |tableaddr| and \[[Values]] set to |values|.
+</div>
+
+<div algorithm>
+    The <dfn constructor for="Table">Table(descriptor)</dfn> constructor, when invoked, performs the following steps:
+    1. Let |n| be |descriptor|.initial.
+    1. If |descriptor|.maximum is present, let |m| be |descriptor|.maximum; otherwise, let |m| be empty.
+    1. If |m| is not empty and |m| &lt; |n|, throw a {{RangeError}} exception.
+    1. Let |type| be the [=table type=] {𝗆𝗂𝗇 n, 𝗆𝖺𝗑 |m|} 𝖺𝗇𝗒𝖿𝗎𝗇𝖼.
+    1. Let |store| be the current agent cluster's [=associated store=].
+    1. Let (|store|, |tableaddr|) be [=alloc_table=](|store|, |type|). <!-- TODO(littledan): Report allocation failure https://github.com/WebAssembly/spec/issues/584 -->
+    1. Set the current agent cluster's [=associated store=] to |store|.
+    1. [=Create a table object=] from the table address |tableaddr| and return the result.
+</div>
+
+<div algorithm>
+    The <dfn method for="Table">grow(d)</dfn> method, when invoked, performs the following steps:
+    1. Let |tableaddr| be the Table instance's \[[Table]] internal slot.
+    1. Let |initialSize| be the length of the Table instance's \[[Values]] internal slot.
+    1. Let |store| be the current agent cluster's [=associated store=].
+    1. Let |result| be [=grow_table=](|store|, |tableaddr|, |d|).
+    1. If |result| is [=error=], throw a {{RangeError}} exception.
+
+        Note: The above exception may happen due to either insufficient memory or an invalid size parameter.
+
+    6. Set the current agent cluster's [=associated store=] to |store|.
+    1. [=Append=] null to the Table instance's \[[Values]] internal slot |d| &minus; |initialSize| times.
```

Comment:
`d` is the number of elements added so I think this should just be `d`, no "minus `initialSize`".

Reply:
Fixed